plasma_framework/contracts/src/exits/payment/PaymentInFlightExitable.sol

```diff
@@ -286,47 +290,22 @@ contract PaymentInFlightExitable is
         return amountIn;
     }
 
-    function startExit(StartExitData memory startExitData, bool isCanonical) private {
-        UtxoPosLib.UtxoPos memory youngestInputUtxo = getYoungestInputUtxo(startExitData.inputUtxosPos);
-
+    function startExit(StartExitData memory startExitData) private {
         PaymentExitDataModel.InFlightExit storage ife = inFlightExits[startExitData.exitId];
         ife.bondOwner = msg.sender;
         ife.exitStartTimestamp = block.timestamp;
-        ife.exitPriority = getInFlightExitPriority(startExitData.exitId, youngestInputUtxo.value, block.timestamp);
         setInFlightExitInputs(ife, startExitData.inputTxs, startExitData.inputUtxosPos);
-        if (!isCanonical) {
-            setNonCanonical(ife);
-        }
-    }
-
-    function getYoungestInputUtxo(UtxoPosLib.UtxoPos[] memory inputUtxosPos) private pure returns (UtxoPosLib.UtxoPos memory) {
-        UtxoPosLib.UtxoPos memory youngest = inputUtxosPos[0];
-        for (uint8 i = 1; i < inputUtxosPos.length; i++) {
-            if (inputUtxosPos[i].value < youngest.value) {
-                youngest = inputUtxosPos[i];
-            }
-        }
-        return youngest;
-    }
-
-    function getInFlightExitPriority(
-        uint192 exitId,
-        uint256 youngestInputUtxoPos,
-        uint256 ethereumBlockTimestamp
-    ) public view returns (uint256) {
-        UtxoPosLib.UtxoPos memory youngestInputUtxo = UtxoPosLib.UtxoPos(youngestInputUtxoPos);
-        (bytes32 _root, uint256 inputUtxoTimestamp) = framework.blocks(youngestInputUtxo.blockNum());
-
-        return ((exitableTimestampCalculator.calculate(ethereumBlockTimestamp, inputUtxoTimestamp, false) << 214)
-            | (youngestInputUtxoPos << 160)) | exitId;
+        setNonCanonical(ife);
```

Comment:
severe: shouldn't it default to canonical?

Reply:
It should, done.